lib/tapioca/trackers/constant_tracker.rb

```diff
@@ -0,0 +1,42 @@
+# typed: true
+# frozen_string_literal: true
+
+require "set"
+
+module Tapioca
+  module Trackers
+    # Registers a TracePoint immediately upon load to track points at which
+    # classes and modules are opened for definition. This is used to track
+    # correspondence between classes/modules and files, as this information isn't
+    # available in the ruby runtime without extra accounting.
+    module ConstantTracker
```

Comment:
`Tapioca::Trackers::ConstantTracker` => `Tracker` seems redundant in this name and the repetition does not bring much value. Will it bring more context to call the module `Tapioca::Trackers::ConstantOrigin`?

And same for the other trackers?

Reply:
Great points! I've done the following renames:
- `Tapioca::Trackers::AutoloadTracker` => `Tapioca::Trackers::Autoload`
- `Tapioca::Trackers::MixinTracker` => `Tapioca::Trackers::Mixin`
- `Tapioca::Trackers::ConstantTracker` => `Tapioca::Trackers::ConstantDefinition` (since this one tracks definition (or redefinition) of constants)